openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -218,12 +218,12 @@ <h3 ng-if="motion.polls.length > 0" translate>Voting result</h3>
   <div class="row">
     <div class="col-sm-8">
       <h3 translate>Text</h3>
-      <div ng-bind-html="motion.getText(version)"></div>
+      <div ng-bind-html="motion.getText(version) | trusted"></div>
```

Comment:
We need this to show style formatting (e.g. font color or background color). I use the [$sce.trustAsHtml ](https://docs.angularjs.org/api/ng/service/$sce) function in my trusted filter.

Note: TinyMCE does not allow to save JS code. Only html5 tags will saved.